packages/scaffold/src/partials/w3m-connecting-siwe/index.ts

```diff
@@ -0,0 +1,52 @@
+import { customElement } from '@web3modal/ui'
+import { LitElement, html } from 'lit'
+import { property } from 'lit/decorators.js'
+import styles from './styles.js'
+
+@customElement('w3m-connecting-siwe')
+export class W3mConnectingSiwe extends LitElement {
+  public static override styles = styles
+
+  // -- State & Properties -------------------------------- //
+  @property() public dappImageSrc?: string
```

Comment:
As this is a partial, it can access these directly, so no need to pass them down and create extra "prop drilling" complexity. properties should mostly be on "simple components".